packages/libs/error-stack/Makefile.toml

```diff
@@ -5,6 +5,10 @@ CARGO_CLIPPY_HACK_FLAGS = "--workspace --feature-powerset --exclude-features def
 CARGO_TEST_HACK_FLAGS_BASE = "--workspace --feature-powerset --exclude-features default --optional-deps eyre,anyhow"
```

Comment:
`CARGO_CLIPPY_HACK_FLAGS` also need adjustment, otherwise it will run 512 combinations 👀

Reply:
fixed in #959

Reply:
Great! 🎉